internal/core/common/ecs/ecs.go

```diff
@@ -0,0 +1,40 @@
+package ecs
+
+import "strings"
+
+type TaskMetadata struct {
+	ClusterName	string `json:"Cluster"`
+	TaskARN		string `json:"TaskARN"`
+	Family		string `json:"Family"`
+	Revision	string `json:"Revision"`
+	KnownStatus	string `json:"KnownStatus"`
+	Containers  []Container `json:"Containers"`
+}
+
+func (task *TaskMetadata) GetDimensions() map[string]string {
+	dims := make(map[string]string, 0)
+	if idx := strings.Index(task.ClusterName, "/"); idx >= 0 {
+		dims["ClusterName"] = task.ClusterName[idx+1: len(task.ClusterName)]
```

Comment:
Match the same dimension names used with the sample config for ECS.

Reply:
Also general naming convention for dimension names is lower case snake_case or kebab-case.